templates/4a_compensate_image_data_round2.ipynb

```diff
@@ -0,0 +1,443 @@
+{
```

Comment:
<p>I would update this to match with the example above, which showed 144/147 contamination</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/319/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>